hardware/Comply-to-the-Regulations.md

```diff
@@ -0,0 +1,394 @@
+
+   - LMA
+     
+     Limit module doesn't meet all 8 criteria sets above for a full module. And it should be used in a known host or known environment. For a licensed limited module, the manufacturer must state how control of the end product into which the module will be installed. It will be maintained that full compliance of the end product is always assured. LMA is not tested as standalone, but instead it will be tested inside a specific host. The certification is valid only for the specific host.
+
+     If the host or end environment is known, Limited Modular Approval may be possible:
+     - Limited to application.
+     - Limited to host.
+     - Limited to installer (typically Grantee or partner).
+
+   - Permissive Changes
+
+     It describes the modifications that may be made to an RF device without new authorization.
+
+     - Class 1 Permissive Change (C1PC);
+       - Changes that do not increase emissions or change RF characteristics. 
+       - Modifications which do not degrade the characteristics reported.  
+       - Does not need application to FCC, i.e. no filing to FCC is required. 
+     Typical C1PC changes: equivalent antenna change or small BOM change.
+     - Class 2 Permissive Change (C2PC); 
+       - Changes that increase emissions or any changes to remove particular restrictions mentioned in the grant. 
+       - Includes those modifications which degrade the performance characteristics as reported however still meet the minimum requirements of the applicable rules.
+       - Official test report required, must be filed with FCC. 
+       Silicon Labs will provide an authorization letter so that the customer can apply for the C2PC on behalf of Silicon Labs. Typical C2PC changes: antenna change with higher gain, antenna type change, co-location of radios, small RF layout changes that result in an electrically equivalent device (can be confirmed by FCC).
+     - Class 3 Permissive Change  (C3PC);
+       - Includes modifications to the software of a software defined radio transmitter that change the frequency range, modulation type or maximum output power(either radiated or conducted) outside the parameters previously ap-proved, or that change the circumstances under which the transmitter operates in accordance with FCC rules
+
+   - New FCC ID and a new equipment authorization application to the FCC required for the following changes:
+     - Frequency multiplication stages;
+     - Basic modulator circuit;
+     - Maximum power or field strength ratings;
+     - Modifications to the RF path / layout that result in a non-electrically equivalent device will require completely new certification. particular change can be confirmed by FCC by submitting a ticket here: https://apps.fcc.gov/oetcf/kdb/forms/InquiryForm.cfm to check what change can be considered as electrically equivalent and thus approved under C2PC;
```

Comment:
Should the url link be inserted for here?  For example:
[here](https://apps.fcc.gov/oetcf/kdb/forms/InquiryForm.cfm)

Reply:
Corrected.